src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Utilities/UsingsAndExternAliasesDirectiveComparer.cs

```diff
@@ -116,7 +116,7 @@ public int Compare(SyntaxNode? directive1, SyntaxNode? directive2)
                 case UsingKind.Alias:
                     var aliasComparisonResult = _tokenComparer.Compare(using1!.Alias!.Name.Identifier, using2!.Alias!.Name.Identifier);
 
-                    if (aliasComparisonResult == 0)
+                    if (aliasComparisonResult == 0 && using1.Name != null && using2.Name != null)
```

Comment:
Should we compare types as well? Or what if one has a name but the other has a type?

Reply:
IDE will make some determinations on what we want the experience to be here.  For now, it's just out of scope and we don't care.  It's def not part of hte minbar work.